chroma_core/models/host.py

```diff
@@ -1816,3 +1816,28 @@ def run(self, kwargs):
         self.invoke_rust_agent_expect_result(
             kwargs["host"], "snapshot_unmount", {"fsname": kwargs["fsname"], "name": kwargs["name"]}
         )
+
+
+class MountSnapshotJob(HostListMixin):
+    fsname = models.CharField(max_length=512)
+    name = models.CharField(max_length=512)
+
+    @classmethod
+    def long_description(cls, stateful_object):
+        return help_text["mount_snapshot"]
+
+    def description(self):
+        if len(self.hosts) > 1:
+            return "Mount snapshot on %d hosts" % len(self.hosts)
+        return "Mount snapshot on host %s" % self.hosts[0]
+
+    def get_steps(self):
+        steps = []
+        for host in self.hosts:
```

Comment:
Would this ever need to run on more than a single host?

Reply:
Scrapped that